problemsets-mat223/linearalbegra.tex

```diff
@@ -4254,13 +4270,75 @@ \section*{Task 2.3: Pat and Jamie}
 	and the non-augmented matrix of coefficients $A=\mat{1&2&1\\1&2&3\\-1&-2&1}$.
 	\begin{parts}
 		\item What is $\Rank(A)$?
+			\begin{solution}
+				$\rref(A)=\mat{1&2&0\\0&0&1\\0&0&0}$, and so the rank of this
+				matrix is 2.
+			\end{solution}
 		\item Give the general solution to \eqref{eq4b}.
+			\begin{solution}
+				$\vec x$ is a solution to the system if 
+				$\vec x=t\mat{-2\\1\\0}$ for some real number $t$.
+
+				If $\vec x=\mat{x\\y\\z}$ is a solution to the system, then we
+				must have $\mat{1&2&0\\0&0&1\\0&0&0}\mat{x\\y\\z}=\mat{x+2y\\z\\0}=\mat{0\\0\\0}$,
+				from which it follows that $z=0$ and $x=-2y$. In other words, 
+				any scalar multiple of $\mat{-2\\1\\0}$ is a solution. 
+			\end{solution}
 		\item Are the column vectors of $A$ linearly independent?
+			\begin{solution}
+				No. The second column is two times the first column.
+			\end{solution}
 		\item Give a non-homogeneous system with the same coefficients as \eqref{eq4b} that has
 			\begin{enumerate}
 				\item infinitely many solutions
 				\item no solutions.
 			\end{enumerate}
+			\begin{solution}
+				\begin{enumerate}
+					\item The following system has infinitely many solutions:
+						\begin{equation}\label{eq4b}
+							\begin{array}{llll}
+								x&+2y&+z &= 1\\
+								x&+2y&+3z &= 1\\
+								-x&-2y&+z &= -1
+							\end{array}
+						\end{equation}
+						We know that for any $\vec x=\mat{x\\y\\z}$, $A\vec x$ is
+						a linear combination of the columns of $A$:
+						\[
+							A\vec x
+							=\mat{1&2&1\\1&2&3\\-1&-2&1}\mat{x\\y\\z}
+							=\mat{x+2y+z\\x+2y+3z\\-x-2y+z}
+							=x\mat{1\\1\\-1}+y\mat{2\\2\\-2}+z\mat{1\\3\\1}. 
+						\]
+						There are infinitely many ways to write $\vec b=\mat{1\\1\\-1}$
+						as a linear combination of the columns of $A$, since for
+						example
+						\[
+							\vec b
+							=\mat{1\\1\\-1}
+							=t\mat{1\\1\\-1}+s\mat{2\\2\\-2}
+							=(t+2s)\mat{1\\1\\-1}
+						\]
+						for any real numbers $s$ and $t$ that satisfy $t+2s=1$.
+						In other words, any $\vec x$ of the form $\mat{1-2s\\s\\0}$
+						is a solution to the system, for any $s \in \R$. 
```

Comment:
The explanation can be dropped. You can just list the systems with no other words. At this point in the course, writing the complete solution should be second nature.